lib/worker.mjs

```diff
@@ -0,0 +1,59 @@
+import { createClient } from 'redis';
+import * as Sentry from '@sentry/node';
+import { nodeProfilingIntegration } from '@sentry/profiling-node';
+
+const isTestEnvironment = process.env.NODE_ENV === 'test';
+const READ_INTERVAL = 3000;
+export const QUEUE_KEY = 'metrics-jobs';
+
+if (!isTestEnvironment) {
+  Sentry.init({
+    debug: true,
+    environment: process.env.NODE_ENV,
+    dsn: 'https://e8c6953f82f7bdadbab491d4fb2c8caf@o4506956365430784.ingest.us.sentry.io/4506956389416960',
+    integrations: [
+      nodeProfilingIntegration(),
+      // ...Sentry.autoDiscoverNodePerformanceMonitoringIntegrations(),
+      new Sentry.Integrations.Postgres(),
+    ],
+    // Performance Monitoring
+    tracesSampleRate: 1.0, //  Capture 100% of the transactions
+    // Set sampling rate for profiling - this is relative to tracesSampleRate
+    profilesSampleRate: 1.0,
+  });
+}
+
+let redisClient;
+export async function init() {
+  redisClient = createClient({
+    url: process.env.REDIS_URL,
+  });
+  redisClient.connect();
+}
+
+export default function consume() {
+  setInterval(async () => {
+    const jobString = await redisClient.lPop(QUEUE_KEY);
+    if (jobString) {
+      const job = JSON.parse(jobString);
+      Sentry.continueTrace(
+        { sentryTrace: job.traceHeader, baggage: job.baggageHeader },
+        (transactionContext) => {
+          Sentry.startSpan({
+            ...transactionContext,
+            name: 'metrics background job',
+            op: 'queue.task',
```

Comment:
This looks good. I need to update our ingestion since we're only processing on `queue.task.celery` and `queue.submit.celery` span ops right now.